packages/react-native/Libraries/AppDelegate/RCTAppDelegate.mm

```diff
@@ -255,7 +255,23 @@ - (RCTRootViewFactory *)createRCTRootViewFactory
   {
     return [weakSelf createBridgeWithDelegate:delegate launchOptions:launchOptions];
   };
+  
+  configuration.sourceURLForBridge = ^NSURL * _Nullable(RCTBridge * _Nonnull bridge) {
+    return [weakSelf sourceURLForBridge:bridge];
+  };
 
+  configuration.extraModulesForBridge = ^NSArray<id<RCTBridgeModule>> * _Nonnull(RCTBridge * _Nonnull bridge) {
+    return [weakSelf extraModulesForBridge:bridge];
+  };
+  
+  configuration.extraLazyModuleClassesForBridge = ^NSDictionary<NSString *,Class> * _Nonnull(RCTBridge * _Nonnull bridge) {
+    return [weakSelf extraLazyModuleClassesForBridge:bridge];
+  };
+  
+  configuration.bridgeDidNotFindModule = ^BOOL(RCTBridge * _Nonnull bridge, NSString * _Nonnull moduleName) {
+    return [weakSelf bridge:bridge didNotFindModule:moduleName];
+  };
+  
```

Comment:
```suggestion
  if ([self respondsToSelector:@selector(extraModulesForBridge:)]) {
    configuration.extraModulesForBridge = ^NSArray<id<RCTBridgeModule>> * _Nonnull(RCTBridge * _Nonnull bridge) {
      return [weakSelf extraModulesForBridge:bridge];
    };
  }

  if ([self respondsToSelector:@selector(extraLazyModuleClassesForBridge:)]) {
    configuration.extraLazyModuleClassesForBridge = ^NSDictionary<NSString *,Class> * _Nonnull(RCTBridge * _Nonnull bridge) {
      return [weakSelf extraLazyModuleClassesForBridge:bridge];
    };
  }

  if ([self respondsToSelector:@selector(bridge:didNotFindModule:)]) {
    configuration.bridgeDidNotFindModule = ^BOOL(RCTBridge * _Nonnull bridge, NSString * _Nonnull moduleName) {
      return [weakSelf bridge:bridge didNotFindModule:moduleName];
    };
  }
```

it may crash in default setup that nobody implements these functions in AppDelegate.

Reply:
Thank you 🙏